### ISSUE TITLE
Exclude patterns

### ISSUE THREAD

**Issue by kvz** [lukeapage/node-markdown-spellcheck#55]
It would be very helpful if I could exclude certain file patterns. 

My usecase is where I spellcheck a Jekyll repo, and I want to check all `**/*.md` files, but I don't want to check any markdown files in `./vendor/` or the generated `./_site/` (we have a few markdown files as-is there too).

I would imagine this too look like

```bash
mdspell --en-us --ignore-numbers --ignore-acronyms **/*.md ./*.md --exclude ./_site --exclude ./vendor
```

**Comment by kvz** [comment]
For those running into this stumbling upon this thread, I'm current working around the issue via:

```bash
mdspell --en-us --ignore-numbers --ignore-acronyms $(./_scripts/find-content.sh)
```

Where `./_scripts/find-content.sh` contains something along the lines of:

```bash
find . \
  -not \( -path ./assets/build -prune \) \
  -not \( -path ./assets/bower -prune \) \
  -not \( -path ./vendor \) \
  -not \( -path ./node_modules -prune \) \
  -not \( -path ./_site -prune \) \
  -name \*.md \
  -o \
  -name \*.markdown \
  -o \
  -name \*.yml
```

**Comment by ColinEberhardt** [comment]
Moving from glob to [globby](https://github.com/sindresorhus/globby) might help here.

**Comment by lukeapage** [comment]
thanks @ColinEberhardt good idea, will look into it..

**Comment by lukeapage** [comment]
Released in 0.9.0

**Comment by kvz** [comment]
Great! :100:
